A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.js

```diff
@@ -304,7 +349,25 @@ class A320_Neo_FCU_Heading extends A320_Neo_FCU_Component {
             this.backToIdleTimeout = 5;
```

Comment:
Magic number, move to constructor. Also, not sure if this should be 45, as per http://www.a320dp.com/A320_DP/nav-autoflight/sys-13.2.8.html